### PR TITLE
Made more style changes, activated scrolling on map, tabs and index default view

### PR DESCRIPTION
**Pull request by straleyb**
Added tabs for ease of transition between map and index view. Also takes away the issue of map being squished after a facet search happens. Basic style changes, made index view default.

### CODE REVIEW

File: app/views/catalog/_search_results.html.erb [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+<% @page_title = t('blacklight.search.title', :application_name => application_name) %>
+
+
+<% content_for(:head) do -%>
+  <%= render_opensearch_response_metadata %>
+  <%= auto_discovery_link_tag(:rss, url_for(params.merge(:format => 'rss')), :title => t('blacklight.search.rss_feed') ) %>
+  <%= auto_discovery_link_tag(:atom, url_for(params.merge(:format => 'atom')), :title => t('blacklight.search.atom_feed') ) %>
+<% end -%>
+<div role='tabpanel'>
+  <ul id="myTab" class="nav nav-tabs" role='tablist'>
```
Comment (terrellt) [inline app/views/catalog/_search_results.html.erb]: Better ID?

Comment (straleyb) [inline app/views/catalog/_search_results.html.erb]: Yessir!

**Comment by terrellt** [comment]
@straleyb If this javascript still works, then that javascript file's totally unecessary, because the divs don't match anymore.

**Comment by straleyb** [comment]
Ok so I kept the javascript in so I could activate the map inside of a tab. I had to change some of the values to match up properly when checking to see if the map isn't loaded yet.

**Pull request merged**